### ISSUE TITLE
Option --tables is not working properly with sysbench-1.0.10 oltp_insert.lua script.

### ISSUE THREAD

**Issue by rameshvs02** [akopytov/sysbench#178]
Option --tables is not working properly with sysbench-1.0.10 oltp_insert.lua script. Tried to create 16 tables but it created only one table.
```
ramesh@qaserver-06:~/pmmwork$ sysbench /usr/share/sysbench/oltp_insert.lua --table-size=100000 --tables=16 --mysql-db=PS_NODE_2 --mysql-user=root  --threads=16 --db-driver=mysql --mysql-socket=/tmp/PS_NODE_2.sock prepare
sysbench 1.0.10 (using bundled LuaJIT 2.1.0-beta2)

Creating table 'sbtest1'...
Inserting 100000 records into 'sbtest1'
Creating a secondary index on 'sbtest1'...
ramesh@qaserver-06:~/pmmwork$
```
oltp_read_write.lua is working fine.

```
ramesh@qaserver-06:~/pmmwork$ sysbench /usr/share/sysbench/oltp_read_write.lua --table-size=100000 --tables=16 --mysql-db=PS_NODE_2 --mysql-user=root  --threads=16 --db-driver=mysql --mysql-socket=/tmp/PS_NODE_2.sock prepare
sysbench 1.0.10 (using bundled LuaJIT 2.1.0-beta2)

Initializing worker threads...

Creating table 'sbtest4'...
Creating table 'sbtest3'...
Creating table 'sbtest5'...
Creating table 'sbtest12'...
Creating table 'sbtest9'...
Creating table 'sbtest15'...
Creating table 'sbtest7'...
Creating table 'sbtest16'...
Creating table 'sbtest14'...
Creating table 'sbtest6'...
Creating table 'sbtest8'...
Creating table 'sbtest10'...
Creating table 'sbtest11'...
Creating table 'sbtest13'...
Creating table 'sbtest2'...
Creating table 'sbtest1'...
Inserting 100000 records into 'sbtest15'
Inserting 100000 records into 'sbtest8'
Inserting 100000 records into 'sbtest7'
Inserting 100000 records into 'sbtest13'
Inserting 100000 records into 'sbtest3'
Inserting 100000 records into 'sbtest10'
Inserting 100000 records into 'sbtest14'
Inserting 100000 records into 'sbtest5'
Inserting 100000 records into 'sbtest4'
Inserting 100000 records into 'sbtest2'
Inserting 100000 records into 'sbtest9'
Inserting 100000 records into 'sbtest6'
Inserting 100000 records into 'sbtest1'
Inserting 100000 records into 'sbtest11'
Inserting 100000 records into 'sbtest12'
Inserting 100000 records into 'sbtest16'
```

**Comment by akopytov** [comment]
Hi @rameshvs02,

This should be fixed in both 1.0 and master branches. You have the following options to get the fix:

- fix oltp_insert.lua manually (it's a simple one-line change, see the linked commit)
- build either 1.0 or the master branch from source
- use pre-release binaries from https://packagecloud.io/akopytov/sysbench-prereleases

**Comment by rameshvs02** [comment]
Thank you @akopytov !
